README.md

```diff
@@ -3,47 +3,37 @@ A fast way to get started with the [Google Maps Android API v2](http://developer
 
 #### Current version of library dependencies:
 
-  * Google Play Services, revision 7
-  * Android Support Library, revision 13
+  * Google Play Services Maps 8.4.0
 
 # TOC
 
   * [Requirements](#requirements)
-  * [Check out the project](#check-out-the-project)
-  * [Eclipse](#eclipse)
-    * [Download Eclipse](#download-eclipse)
-    * [Import the project to your workspace](#import-the-project-to-your-workspace)
-    * [Add your API key](#add-your-api-key)
-    * [Run the app](#run-the-app)
-    * [Troubleshooting](#troubleshooting)
-  * [IntelliJ](#intellij)
-    * [Download IntelliJ](#download-intellij)
-    * [Import the project](#import-the-project)
+  * [Android Studio](#android-studio)
+    * [Download Android Studio](#download-android-studio)
+    * [Open the project](#open-the-project)
     * [Add your API key](#add-your-api-key-1)
     * [Run the app](#run-the-app-1)
     * [Troubleshooting](#troubleshooting-1)
 
 # Requirements
 
-Java Development Kit (JDK) [Download](http://www.oracle.com/technetwork/java/javase/downloads/jdk7-downloads-1880260.html)
+Java Development Kit (JDK) [Download](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
 
 Android SDK [Download](http://developer.android.com/sdk/index.html)
 
-# Eclipse
+# Android Studio
 
-## Download Eclipse
-Eclipse is packaged with the [Android Developer Tools bundle](http://developer.android.com/sdk/index.html)
+## Download Android Studio
+Download Android Studio (http://developer.android.com/sdk/index.html)
 
-## Import the project to your workspace
-  1. Choose File> Import...
+## Open the project
+  1. On the welcome screen, select "Open an existing Android Studio project" or if you have a project open, choose "File->Open...""
   2. Choose Android > Existing Android Code Into Workspace
   3. Navigate to the checked out project
-  4. You should see two projects - "hellomap" and "google-play-services_lib".
-  5. Ensure both are selected
-  6. Press "Finish"
+  4. Press "Ok"
 
 ## Add your API key
```

Comment:
This section doesn't look right for Android Studio. in the default 'Android' view, the manifest is under 'Application -> manifests'